<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqApplicationOptions</class>
 <widget class="QWidget" name="pqApplicationOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="General">
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Default View</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="DefaultViewType"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Heart-Beat Interval</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="HeartBeatTimeout">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;p&gt;Set the interval in &lt;b&gt;minutes&lt;/b&gt; after which the client sends a heart beat message to the server to avoid connection termination due to inactivity. &lt;b&gt;0&lt;/b&gt; means no heartbeats.&lt;/p&gt;&lt;/html&gt;</string>
         </property>
         <property name="statusTip">
          <string>&lt;html&gt;&lt;p&gt;Set the interval in &lt;b&gt;minutes&lt;/b&gt; after which the client sends a heart beat message to the server to avoid connection termination due to inactivity. &lt;b&gt;0&lt;/b&gt; means no heartbeats.&lt;/p&gt;&lt;/html&gt;</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;p&gt;Set the interval in &lt;b&gt;minutes&lt;/b&gt; after which the client sends a heart beat message to the server to avoid connection termination due to inactivity. &lt;b&gt;0&lt;/b&gt; means no heartbeats.&lt;/p&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>mins</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Auto Accept</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="AutoAccept">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Crash recovery</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="CrashRecovery">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="LabelMultiCore">
         <property name="text">
          <string>Use Multi-Core</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="AutoMPI">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>On File Open</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="DefaultTimeStepMode">
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Helvetica'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Control the timestep shown by ParaView by default when a dataset with multiple timesteps is opened.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Goto current timestep</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Goto last timestep</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Rescale Data Range Mode</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="RescaleDataRangeMode">
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Helvetica'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Controls how the ranges for scalar mapping to colors is adjusted.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Update on Apply</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Grow every timestep</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="toolTip">
          <string>Automatic conversion of cell to point and point to cell properties.</string>
         </property>
         <property name="text">
          <string>Auto Convert Properties</string>
         </property>
         <property name="buddy">
          <cstring>AutoConvertProperties</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="AutoConvertProperties">
         <property name="toolTip">
          <string>Automatic conversion of cell to point and point to cell properties.</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="StrictLoadBalancing">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="StrictLoadBalancingLabel">
         <property name="text">
          <string>Use Strict Load Balancing</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="SpecularHighlightingLabel">
         <property name="text">
          <string>Specular Highlighting with Scalar Coloring</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="SpecularHighlighting">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="DisableSplashScreenLabel">
         <property name="text">
          <string>Disable Splash Screen</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QCheckBox" name="DisableSplashScreen">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Colors">
      <layout class="QVBoxLayout">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Helvetica'; font-size:9pt;&quot;&gt;These are color categories. When assigning colors to objects, one can select one of these catergories. In that case if color is changed in this dialog, then all colors assigned to the same category are updated automatically.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="pqColorChooserButton" name="ForegroundColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Foreground Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="pqColorChooserButton" name="BackgroundColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Background Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="pqColorChooserButton" name="SurfaceColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Surface Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="pqColorChooserButton" name="TextAnnotationColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Text Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="pqColorChooserButton" name="SelectionColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Selection Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="pqColorChooserButton" name="EdgeColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Edge Color</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QToolButton" name="Palette">
           <property name="text">
            <string>Choose Palette</string>
           </property>
           <property name="popupMode">
            <enum>QToolButton::MenuButtonPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>53</width>
             <height>28</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ResetColorsToDefault">
           <property name="text">
            <string>Reset to Default</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>364</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Animation">
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="AnimationCacheGeometry">
         <property name="toolTip">
          <string>Cache geometry during playback.</string>
         </property>
         <property name="text">
          <string>Cache Geometry</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="AnimationCacheLimitLabel">
         <property name="toolTip">
          <string>&lt;html&gt;Set the cache limit (in KBs) per node. If while playing animations, the cache size on any node goes beyond this threshold, caching will automatically be disabled.&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Cache Limit (in KBs)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="AnimationCacheLimit">
         <property name="toolTip">
          <string>&lt;html&gt;Set the cache limit (in KBs) per node. If while playing animations, the cache size on any node goes beyond this threshold, caching will automatically be disabled.&lt;/html&gt;</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>358</width>
           <height>101</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Charts">
      <layout class="QGridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="HidenSeriesLabel">
           <property name="text">
            <string>Hidden Series</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="ChartNewHiddenSeries">
           <property name="toolTip">
            <string>Add a regular expression to the hidden series list.</string>
           </property>
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="ChartDeleteHiddenSeries">
           <property name="toolTip">
            <string>Remove the selected regular expression from the hidden series list.</string>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>118</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" rowspan="4">
          <widget class="QListWidget" name="ChartHiddenSeries">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>By default, all series in a data set are initially shown in a chart except those with names that match the regular expressions in this list.</string>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="ChartResetHiddenSeries">
           <property name="toolTip">
            <string>Reset the hidden series list to the defaults that come with ParaView.</string>
           </property>
           <property name="text">
            <string>Reset to Default</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>116</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqColorChooserButton</class>
   <extends>QPushButton</extends>
   <header>pqColorChooserButton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>DefaultViewType</tabstop>
  <tabstop>DefaultTimeStepMode</tabstop>
  <tabstop>RescaleDataRangeMode</tabstop>
  <tabstop>HeartBeatTimeout</tabstop>
  <tabstop>AutoAccept</tabstop>
  <tabstop>AutoConvertProperties</tabstop>
  <tabstop>CrashRecovery</tabstop>
  <tabstop>AutoMPI</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>ForegroundColor</tabstop>
  <tabstop>BackgroundColor</tabstop>
  <tabstop>SurfaceColor</tabstop>
  <tabstop>TextAnnotationColor</tabstop>
  <tabstop>SelectionColor</tabstop>
  <tabstop>EdgeColor</tabstop>
  <tabstop>Palette</tabstop>
  <tabstop>ResetColorsToDefault</tabstop>
  <tabstop>AnimationCacheGeometry</tabstop>
  <tabstop>AnimationCacheLimit</tabstop>
  <tabstop>ChartNewHiddenSeries</tabstop>
  <tabstop>ChartDeleteHiddenSeries</tabstop>
  <tabstop>ChartHiddenSeries</tabstop>
  <tabstop>ChartResetHiddenSeries</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
